README.md

```diff
@@ -172,30 +172,20 @@ If we already have a WMS GetMap URL, we can use it directly:
 Requests to Processing API need to be authenticated.
 Documentation about authentication is available at [Sentinel Hub documentation](https://docs.sentinel-hub.com/api/latest/#/API/authentication).
 
-In short, authentication is done by requesting an authentication token and setting it as described in [Layers](#layers).
-The authentication token is retrieved by making a request to the `https://services.sentinel-hub.com/oauth/token` endpoint with the OAuth Client's id and secret.
+In short, authentication is done by getting an authentication token using OAuth Client's id and secret, and setting it.
 
 To get the OAuth Client's id and secret, a new OAuth Client must be created in [**User settings**](https://apps.sentinel-hub.com/dashboard/#/account/settings) on **Sentinel Hub Dashboard** under **OAuth clients**.
 OAuth Client's secret is shown only before the creation process is finished so be mindful to save it.
 
-In javascript requesting the authentication token can be done with builtin XMLHttpRequest:
+Getting the authentication token by calling `getAuthTokenFromOAuth()` with the OAuth Client's id and secret as its parameters and then setting the authentication token:
```

Comment:
I know it was my suggestion, but maybe this name is not really perfect (technically, we are getting the token from OAuth _service_)... Not sure I have a better one though - let's discuss live.

Reply:
We talked live and agreed to name this function `requestAuthToken` as it tells more about what the function does.

Will rename it.

Reply:
Did rename it and updated the examples and readme.